concepts.wip/boolean-logic/introduction.md

```diff
@@ -1,57 +1,62 @@
 # Introduction
 
-Imagine we have the following Boolean expressions in Julia: `5 > 3` and `1 != 0`, both of the expression are logically `true`. How can we combine the previously mentioned expressions to produce a `true` or `false` result?
-
-We use [logical connectives](https://en.wikipedia.org/wiki/Logical_connective)! Also called _logical operators_. In Julia, they are called [Boolean operators](https://docs.julialang.org/en/v1/manual/mathematical-operations/#Boolean-Operators).
+In Julia, according to the [operator precedence](https://docs.julialang.org/en/v1/manual/mathematical-operations/#Operator-Precedence-and-Associativity), `&&` has higher prority over `||`.
 
-Below, we provide examples on how to use them to produce the desired result.
+Imagine we have the following Boolean expressions in Julia: `5 > x` and `x != 0`.
+If `x` was 3 they would both be `true`.
+We can express statements like "is x less than 5 and not equal to y?" using [Boolean operators](https://docs.julialang.org/en/v1/manual/mathematical-operations/#Boolean-Operators): `!` (not), `&&` (and), `||` (or).
 
-## Logical _and_
+### Logical _not_
```

Comment:
I think the level of the heading is wrong here, though I'm not up-to-date on Exercism's markdown conversion spec. Disregard if that was intentional.

Reply:
Goes back to `## Boolean logic`. I assumed `### Logical _not` is a level under that.

Reply:
```suggestion
## Logical _not_
```